web/tasks/DART.md

```diff
@@ -10,110 +11,175 @@ motivation: TODO.
+| ------------- | ------------- || ------------- || ------------- || ------------- || ------------- |
+|Triple Set | 82,191 | 33.2K | 21.6 | 1.5 | 5,623 |
 
-### Data Statistics
+| Train | Dev | Test|
+| ------------- | ------------- || ------------- |
+| 62,659 | 6,980 | 12,552|
 
-[More Information Needed]
 
-## Dataset Creation
+Statistics of DART decomposed by different collection methods. DART exhibits a great deal of topical variety in terms of the number of unique predicates, the number of unique triples, and the vocabulary size. These statistics are computed from DART v1.1.1; the number of unique predicates reported is post-unification (see Section 3.4). SR: Surface Realization.
+([details in Table 1 and 2](https://arxiv.org/pdf/2007.02871.pdf)).
 
-### Curation Rationale
 
-[More Information Needed]
+## Dataset Creation 
 
-### Communicative Goal
+### Curation Rationale
+
+The dataset creators encourage through DART further research in natural language generation from semantic data.
+DART provides high-quality sentence annotations with each input being a set of entity-relation triples in a tree structure. 
 
-[More Information Needed]
+### Communicative Goal 
 
+The speaker is required to produce coherent sentences and construct a trees structured ontology of the column headers.
 
 ### Source Data
 
+The dataset re-uses data from the following pre-existing resources:
+(1) human annotation on open-domain Wikipedia tables from WikiTableQuestions ([Pasupat and Liang,
+2015](https://www.aclweb.org/anthology/P15-1142.pdf)) and WikiSQL ([Zhong et al., 2017](https://arxiv.org/pdf/1709.00103.pdf))
+(2) automatic conversion of questions in
+WikiSQL to declarative sentences
+(3) incorporation of existing datasets including WebNLG 2017 (Gardent et al., 2017[a](https://www.aclweb.org/anthology/P17-1017.pdf),[b](https://www.aclweb.org/anthology/W17-3518.pdf); [Shimorina and Gardent, 2018](https://www.aclweb.org/anthology/W18-6543.pdf)) and Cleaned E2E ([Novikova et al., 2017b](https://arxiv.org/pdf/1706.09254.pdf); Dušek et al., [2018](https://arxiv.org/pdf/1810.01170.pdf), [2019](https://www.aclweb.org/anthology/W19-8652.pdf))
+
+Creators also explored automatic alignments between the knowledge base and text including Neural Wikipedian ([Vougiouklis et al., 2018](https://arxiv.org/pdf/1711.00155.pdf)) and TRex ([Elsahar et al., 2018](https://www.aclweb.org/anthology/L18-1544.pdf)).
+
+We refer the reader to the papers describing these sources for further information.
+
 #### Initial Data Collection and Normalization
 
-[More Information Needed]
+The training data consists of concept sets and captions for the source datasets listed above.
+For conversion of a meaning representation (MR) to a triple set, where the NAME slot was represented as the subject.
 
 #### Who are the source language producers?
 
-[More Information Needed]
+Human annotations and an automatic conversion procedure.
```

Comment:
Can you note here whether information about this process has been provided? See https://gem-benchmark.github.io/tasks/guide#who-are-the-source-language-producers